apps/extension/src/screens/Dashboard/index.tsx

```diff
@@ -1,17 +1,12 @@
 import { FC } from 'react';
-import { Link } from 'react-router-dom';
-import { Text } from '@walless/ui';
 
+import ChooseLayout from './ChooseLayout';
 import DashboardLayout from './Layout';
 
 export const HomeScreen: FC = () => {
 	return (
 		<DashboardLayout contentContainerClass="bg-gradient-to-b from-color-5 to-color-6 flex-1">
-			<Text className="text-white text-5xl mb-3">COMING SOON!</Text>
-			<Text>
-				<Text className="text-gray-400">Created with ❤️ by Metacraft</Text>
-			</Text>
-			<Link to="login">Login</Link>
+			<ChooseLayout />
 		</DashboardLayout>
```

Comment:
Use self-closing syntax for empty Tag.
```typescript
<DashboardLayout ... />
```

Reply:
Hi leader, I don't get the problem. There is a `<ChooseLayout />` (with self-closing syntax already) inside the `<DashboardLayout> <ChooseLayout /> </DashboardLayout>`.